tests/cli/keyboard_interface.c

```diff
@@ -50,7 +44,7 @@ int connect_tcp() {
 
     sockfd = socket(AF_INET, SOCK_STREAM, 0);
     if (sockfd == -1) {
-        printf("socket creation failed...\n");
+        printf("Keyboard interface: socket creation failed...\n");
```

Comment:
Just a convention thing--error messages should follow the format
`"<function>: <error message>"`
This is so we know exactly in which function caused the error.
So maybe `"connect_tcp: socket creation failed in Keyboard interface"` or similar?

Just realized we didn't add this to the Wiki yet hm

Reply:
the convention is in the style guide i think